### PR TITLE
Supplier case study form

### PR DESCRIPTION
**Pull request by richtier**
requires these to be merged first:

 - [x] https://github.com/uktrade/directory-validators/pull/17
 - [x] https://github.com/uktrade/directory-api-client/pull/19
 - [x] https://github.com/uktrade/directory-api/pull/70

# step one
image

# step two

image

### CODE REVIEW

File: company/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    )
+
+
+@pytest.fixture
+def supplier_case_study_end_to_end(
+    client, supplier_case_study_basic_data, supplier_case_study_rich_data
+):
+    # loop over each step in the supplier case study wizard and post valid data
+    data_step_pairs = [
+        [SupplierCaseStudyView.BASIC, supplier_case_study_basic_data],
```
Comment (monikasulik) [inline company/tests/test_views.py]: Hmmm, this bit is a a bit risky, no? If a new step were to be added, somebody could miss that this test needs to be updated.

**Comment by monikasulik** [comment]
None of the above comments are a big deal btw, so react to them as you feel fit.

File: company/forms.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+
+PLEASE_SELECT = 'Please select an option'
+
+
+class CaseStudyBasicInfoForm(forms.Form):
+    title = forms.CharField(
+        max_length=100,
+    )
+    description = forms.CharField(
```
Comment (richtier) [inline company/forms.py]: there is no such thing :) on models there is `TextField`, but not on forms.

Comment (monikasulik) [inline company/forms.py]: Aaah, that makes sense ;) But shouldn't this have a textarea widget thingie on this then?

**Pull request merged**